src/components/Search/SearchComponent.tsx

```diff
@@ -94,10 +94,7 @@ function SearchComponent({
           styles.dropDown
         } label-edit-wrapper`}
       >
-        <InputLabel
-          id='select-repository-label'
-          className='dropdown-select-label'
-        >
+        <InputLabel id={selectLabel} className='dropdown-select-label'>
```

Comment:
Same remark as below about the id. In this was this works because this is just one word, but if we change it in the future this might lead to problems.
I understand you wanted to make it unique. But maybe we can just remove it. Do we use it anywhere?
What do you think?

Reply:
Same reply as above.